terraform/azure_rke2_cluster/examples/gmsa.tfvars

```diff
@@ -45,14 +44,14 @@ apps = {
   }
 
   gmsa-crd = {
-    path         = "https://charts.rancher.io/assets/rancher-windows-gmsa-crd/rancher-windows-gmsa-crd-2.0.0.tgz"
+    path         = "https://github.com/HarrisonWAffel/charts/raw/update-gmsa/assets/rancher-windows-gmsa-crd/rancher-windows-gmsa-crd-3.0.0.tgz"
```

Comment:
all of these forks need to be modified, this PR will be blocked from merge until things are in official repositories.